docs/source/getting-started/setting-up-driver-plugin.adoc

```diff
@@ -77,6 +77,33 @@ $ sudo usermod -a -G libvirt <username>
 $ newgrp libvirt
 ----
 
+[[kvm-driver-arch-linux]]
+=== On Arch Linux
+
+. Install *libvirt*, *qemu*, *dnsmasq* and *ebtables* on your system:
++
+----
+$ sudo pacman -S libvirt qemu dnsmasq ebtables
+----
+
+. Add yourself to *kvm* and *libvirt* groups:
++
+----
+$ sudo usemod -a -G kvm,libvirt <username>
```

Comment:
@anjannath for us it something like below, this looks like a long pending PR, can you do a check + rebase so we can merge it?

```
$ sudo groupmems -g kvm -l
qemu 

$ sudo groupmems -g libvirt -l
prkumar 
```

Reply:
@praveenkumar Sure.. on it now!

Reply:
```
➜  ~ sudo groupmems -g kvm -l
anath 
➜  ~ sudo groupmems -g libvirt -l
anath 
```
there is no `qemu` user present in Arch Linux by default.